doc/tutorials/tutorial_X8.rst

```diff
@@ -176,6 +178,51 @@ counts:
 >>> counts[(0, 0, 0, 0, 0, 0, 0)]
 2
 
+Job management
+~~~~~~~~~~~~~~
+
+Above, when we called ``eng.run()``, we had to wait for the
+remote device to execute the program and the result to be returned before we could
+continue executing code. That is, ``eng.run()`` is a **blocking** method.
+
+Sometimes, however, it is useful to submit the program
+and continue performing computation locally, every now and again checking to see
+if the job is complete and the results are ready. This is possible with
+the **non-blocking** :meth:`eng.run_async() <.RemoteEngine.run_async>` method:
+
+>>> job = engine.run_async(program, shots=100)
+
+This method is complete as soon as the job has been submitted; no need to wait for
+the job to move through the queue, execute, and for the results to be returned.
```

Comment:
A bit of confusing phrasing. You still need to wait for these things before you see your results, you're just not blocked at the prompt until then

Reply:
Yes, maybe just remove this sentence. I think it's clear what's going on